extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -111,12 +161,17 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 				CSSStyles: { 'padding-top': '0' }
 			}).component();
 
+		tasksContainer.addItem(form, {});
```

Comment:
nit : empty second parameter might not be required

Reply:
This is the pattern followed throughout the code where we have multiple items in the same row, also this just adds a empty option item layout which is of no harm, so keeping it the same for consistency.